src/JsonSchema/Uri/UriRetriever.php

```diff
@@ -22,12 +22,23 @@
  */
 class UriRetriever
 {
+    /**
+     * @var null|UriRetrieverInterface
+     */
     protected $uriRetriever = null;
 
     /**
+     * @var array|object[]
```

Comment:
follows this projects convention.

no need for short descr, @see + varname, no waste needed.